git-transport/src/client/blocking_io/http/reqwest.rs

```diff
@@ -19,16 +24,27 @@ mod remote {
             let (req_send, req_recv) = std::sync::mpsc::sync_channel(0);
             let (res_send, res_recv) = std::sync::mpsc::sync_channel(0);
             let handle = std::thread::spawn(move || -> Result<(), reqwest::Error> {
```

Comment:
One option regarding errors would be to change the error type here to something more general.